src/socketio.quiver.js

```diff
@@ -407,14 +457,31 @@ QuiverSocialProvider.prototype.sendAllRosterChanged_ = function() {
   }
 };
 
-QuiverSocialProvider.prototype.countOwnerConnections_ = function() {
-  var count = 0;
-  for (var serverKey in this.connections_) {
-    if (this.connections_[serverKey].owner) {
-      ++count;
+/**
+ * @return {!Array.<!QuiverSocialProvider.server_>} The intersection of servers
+ *     that I am connected to and servers that I am advertising in my intro
+ *     message.  Includes connections that have not yet succeeded.
```

Comment:
Can you comment on why we would want to advertise servers that have not yet succeed in connecting?  I see in the inviteUser code that we don't actually include a "live advertised server" unless it is also connected (connection.ready has fulfilled) - are we sure we still want to include those servers in the intro message?